test/RibbonOptionsVault.js

```diff
@@ -84,6 +93,54 @@ describe("RibbonCoveredCall", () => {
     tokenDecimals: 8,
     isPut: false,
   });
+
+  behavesLikeRibbonOptionsVault({
+    name: `Ribbon WBTC Theta Vault (Put)`,
+    tokenName: "Ribbon BTC Theta Vault Put",
+    tokenSymbol: "rWBTC-THETA-P",
+    asset: WBTC_ADDRESS,
+    assetContractName: "IUSDC",
+    optionType: PUT_OPTION_TYPE,
+    strikeAsset: USDC_ADDRESS,
+    collateralAsset: USDC_ADDRESS,
+    wrongUnderlyingAsset: WETH_ADDRESS,
+    wrongStrikeAsset: WETH_ADDRESS,
+    firstOptionStrike: 2400,
+    secondOptionStrike: 2500,
+    chainlinkPricer: CHAINLINK_WBTC_PRICER,
+    tokenDecimals: 18,
+    depositAmount: BigNumber.from("100000000"),
+    premium: BigNumber.from("10000000"),
+    minimumSupply: BigNumber.from("10").pow("3").toString(),
+    isPut: true,
```

Comment:
We can remove `isPut` since we already have `optionType` in here. So we can remove one or the other